Cyclismo/Cyclismo/src/main/java/org/cowboycoders/cyclismo/turbo/TurboService.java

```diff
@@ -79,13 +80,25 @@
 
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.TimeoutException;
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
 public class TurboService extends Service {
 
+  // Available turbos.
+  // TODO: Clean this up: Register turbos with TurboService, which builds a list of available turbos?
+  private final String BUSHIDO_HEADUNIT = "TACX_BUSHIDO_HEADUNIT"; //getString(R.string.turbotrainer_tacx_bushido_headunit_value);
```

Comment:
Just checking you were paying attention ;) I was hoping to get rid of these, and use the ```modes``` provided by ```GenericTurboTrainer```. Was that your original intention? Then maybe a have a Turbo registry, which can return all the Turbos and their modes.